manifests/profiles/default.yaml

```diff
@@ -68,8 +68,8 @@ spec:
               targetPort: 15443
               name: tls
         hpaSpec:
-          maxReplicas: 5
-          minReplicas: 1
+          maxReplicas: 10
```

Comment:
I don't think there is a good reason to change maxReplicas, especially here.

Reply:
Ok, I can remove the changes to `maxReplicas` everywhere then. Per my review notes, I wasn't sure if that was necessarily warranted.

Reply:
Done